purchase_manual_service_quantity/README.rst

```diff
@@ -0,0 +1,56 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+================================
+Purchase Manual Service Quantity
+================================
+
+Allow to Manually set received quantities for product service on purchase order
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Configure service product to allow to proceed to the manual reception of the quantities directly on the order form
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/142/10.0
+
+Bug Tracker
```

Comment:
pls, update READM according to latest https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst